libqtile/widget/launchbar.py

```diff
@@ -210,10 +213,11 @@ def draw(self):
                 # display an icon
                 self.drawer.ctx.set_source(self.surfaces[name])
                 self.drawer.ctx.paint()
-            self.drawer.draw(self.offset + xoffset, icon_width + self.padding)
+            self.drawer.draw(offsetx=self.offset + xoffset,
+                             width=icon_width + self.padding)
             xoffset += icon_width + self.padding
 
-    def calculate_width(self):
+    def calculate_length(self):
```

Comment:
Oh, I was thinking of keeping the code the same, just renaming the method so that it's "obvious" that it's not a purely vertical or horizontal property (indeed, aren't you already just using this method and doing the right thing based on the widget's configured orientation?). The only name I could come up with is `calculate_size()`, and I'm not sure that's really better than this.